db/version_edit.cc

```diff
@@ -59,6 +59,7 @@ enum Tag : uint32_t {
   kDbId,
   kBlobFileAddition,
   kBlobFileGarbage,
+  kSafeToIgnore,
```

Comment:
Sorry for the confusion during offline communication. I suggest that we rename it to something like `kStateUponManifestSwitch`.